imageroot/pypkg/get_route.py

```diff
@@ -79,6 +83,10 @@ def get_route(data):
         if route.get("path"):
             route['strip_prefix'] = True if middlewares and f'{module}-stripprefix@redis' in middlewares else False
 
+        # Check if the route was created manually
+        r = agent.redis_connect(privileged=True)
+        route['user_created'] = r.sismember(f'{agent_id}/user_created_routes', data["instance"])
```

Comment:
Single-char variable names are hard to search in the code. :smile_cat: please type more chars!

```suggestion
        rdb = agent.redis_connect(privileged=True)
        route['user_created'] = rdb.sismember(f'{agent_id}/user_created_routes', data["instance"])
```

Furthermore variable `r` is already used in the scope for multiple regexp match assignments. ..What about renaming it too and using more variables? e.g. `host_match`, `path_match`...

Reply:
I will change the variables' names outside of this PR.